cmd/config/api/api.go

```diff
@@ -50,6 +51,7 @@ const (
 	EnvAPISecureCiphers            = "MINIO_API_SECURE_CIPHERS"
 	EnvAPIReplicationWorkers       = "MINIO_API_REPLICATION_WORKERS"
 	EnvAPIReplicationFailedWorkers = "MINIO_API_REPLICATION_FAILED_WORKERS"
+	EnvAPIS3Zip                    = "MINIO_API_S3ZIP"
```

Comment:
I personally think we should actually do this like a bucket configuration such as add a new API just like BucketQuota API And use perhaps use special tags on a bucket to enable this feature? 

This would allow for bucket-level changes and toggle as well, this ZIP support can also turn on and off the `tar.gz` untar feature as well. 

I am going to check on this with @abperiasamy